src/envVariables.js

```diff
@@ -181,6 +181,10 @@ let environmentVariables = {
 		message: 'Interface Service host address',
 		optional: false,
 	},
+	OBSERVATION_DEEP_LINK_REGEX: {
```

Comment:
@adithyadinesh0412  if its optional gave default value

Reply:
what should be the default value? @priyanka-TL  In each env this can change. 
For ex :
dev : "^https:\/\/dev\.elevate-ml\.shikshalokam\.org\/view\/observation\/[a-f0-9]{32}$"
qa : "^https:\/\/qa\.elevate-ml\.shikshalokam\.org\/view\/observation\/[a-f0-9]{32}$"
etc...

Reply:
ok then make it mandatory

Reply:
what if the adopter don't have observation capability? Only adopting projects? @priyanka-TL

Reply:
OK then they can enable the observation with an env key; if that observation task is ON, then they can add this key as mandatory.